dkulib/dku_config/dss_parameter.py

```diff
@@ -40,6 +42,10 @@ def __init__(self, name: str, value: Any, checks: List[dict] = None, required: b
         self.required = required
         self.cast_to = cast_to
         self.checks = [CustomCheck(**check) for check in checks]
+        if label is None:
+            self.label = name
+        else:
+            self.label = label
```

Comment:
```suggestion
        self.label = label or name
```
To make it more pythonic but if you think it is less readable, leave it that way, it works perfectly!

Reply:
done in https://github.com/dataiku/dss-plugin-dkulib/pull/17/commits/ffaa0fc4a91aeb0ffb4d10413ed835bf4b5097ce